homeworkArraysFunctions.ts

```diff
@@ -0,0 +1,114 @@
+//1.	 Create an array of strings and add a new element at the end of the array. Log the result.
+
+let bookGenre: string[] = ["Criminal", "Detective","Romance","Historical"];
+bookGenre.push("Horror");
+
+console.log("bookGenre", bookGenre);
+
+
+//2.	Create an array of numbers and remove the first element from the array. Log the result.
+
+let numberArray: number[] = [15,20,25,30,35];
+numberArray.pop();
+console.log("numberArray", numberArray);
+
+//3.	 Use the map method to create a new array and divide each number by 2  “num / 2”   from [1, 2, 3, 4, 5].  Log the result.
+
+let baseNumArr: number []= [1,2,3,4,5];
+function numdivision(element:number) :number {
```

Comment:
You don't need a function. Just map the array with method map and divide the numbers:
baseNumArr.map(num => num / 2)